runtime/src/accounts_index.rs

```diff
@@ -1574,24 +1768,33 @@ pub mod tests {
     fn compare(hashset: &HashSet<u64>, bitfield: &RollingBitField) {
         assert_eq!(hashset.len(), bitfield.len());
         assert_eq!(hashset.is_empty(), bitfield.is_empty());
-        let mut min = Slot::MAX;
-        let mut max = Slot::MIN;
-        for item in bitfield.get_all() {
-            min = std::cmp::min(min, item);
-            max = std::cmp::max(max, item);
-            assert!(hashset.contains(&item));
-        }
-        assert!(
-            bitfield.range_width()
-                >= if bitfield.is_empty() {
+        if !bitfield.is_empty() {
+            let mut min = Slot::MAX;
+            let mut max = Slot::MIN;
+            for item in bitfield.get_all() {
```

Comment:
does it make sense to also assert this: `assert_eq!(bitfield.get_all().len(), hashset.len())`